### PR TITLE
virtualFields for custom displayField + Accessor != Mutator

### PR DESCRIPTION
**Pull request by inoas**
Ref: https://book.cakephp.org/3.0/en/orm/entities.html#accessors-mutators
Ref: https://github.com/cakephp/cakephp/issues/10175#issuecomment-277534228

### CODE REVIEW

File: en/orm/retrieving-data-and-resultsets.rst [hunk 1]
```diff
@@ -285,20 +285,35 @@ You can also create list data from associations that can be reached with joins::
...
-Lastly it is possible to use closures to access entity mutator methods in your
-list finds. This example shows using the ``_getFullName()`` mutator method from
+Customize Key-Value Output
+--------------------------
+
+Lastly it is possible to use closures to access entity accessor methods in your
+list finds. ::
+
+    // In your Authors Entity create a virtual field to be used as the displayField:
+    function _getLabel()
```
Comment (antograssiot) [inline en/orm/retrieving-data-and-resultsets.rst]: Missing visibility

File: en/orm/retrieving-data-and-resultsets.rst [hunk 2]
```diff
@@ -285,20 +285,35 @@ You can also create list data from associations that can be reached with joins::
...
+    {
+        return __('Name %s', $this->_properties['name'])
+          . ' / ' . __('User ID %s', $this->_properties['category_id']);
```
Comment (antograssiot) [inline en/orm/retrieving-data-and-resultsets.rst]: User ID with category_I'd feels strange.
I don'the really see the advantage of this example...I just feel it is likely to confuse people

Comment (inoas) [inline en/orm/retrieving-data-and-resultsets.rst]: My bad. Fixing.

**Comment by inoas** [comment]
Anything left to be done?

**Pull request merged**